prov/verbs/src/fi_verbs.c

```diff
@@ -595,6 +604,30 @@ static int fi_ibv_read_params(void)
 			   "Invalid value of use_odp\n");
 		return -FI_EINVAL;
 	}
+
+	if (fi_ibv_get_param_bool("use_xrc", "Enable XRC transport for message "
+				  "endpoint. XRC is under development.",
+				  &fi_ibv_gl_data.msg.use_xrc)) {
```

Comment:
Is the "XRC is under development" statement still true?

Reply:
I think so, at least from a maturity standpoint. However, I'm fine to remote the last sentence if that is preferred by all.